daemon/oci_linux.go

```diff
@@ -486,6 +486,7 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		userMounts[m.Destination] = struct{}{}
 	}
 
+	parser := volume.NewParser("linux")
```

Comment:
nit: move this down to line 511 so it's next to the code that uses it, instead of stranded up here near unrelated code